native/CMakeLists.txt

```diff
@@ -438,6 +523,19 @@ cocos_source_files(
     cocos/platform/interfaces/modules/ISystemWindow.h
     cocos/platform/interfaces/modules/IVibrator.h
 )
+endif()
+
+if(OPENHARMONY)
```

Comment:
`else()` ?

Reply:
It's temporary at the moment and will open later